phonegap-nfc/phonegap-nfc-tests.ts

```diff
@@ -97,7 +97,7 @@ let eventTarget: EventTarget = {
     removeEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean) { return; }
 }
 
-let ndefTagEvent: NdefTagEvent = {
+let ndefTagEvent = {
```

Comment:
the shape of event changes based on Event in lib.d.ts, and TS 2.1 and TS 2.0 has different properties. with the original test one of them would fail, either excess property checks on TS 2.0 or assignablity on TS 2.1.

this way the test should pass on both.